src/pages/About.stories.tsx

```diff
@@ -1,21 +1,13 @@
-import React from 'react'
 // also exported from '@storybook/react' if you can deal with breaking changes in 6.1
 import { Meta } from '@storybook/react/types-6-0'
 
 import About from 'pages/About'
-import GlobalStyles from 'styles/global'
+import { PageDecorator } from 'storybook/PageDecorator'
 
 export default {
-  title: 'Mesa/About',
+  title: 'Pages/About',
   component: About,
-  decorators: [
-    (Story): JSX.Element => (
-      <>
-        <GlobalStyles />
-        <Story />
-      </>
-    ),
-  ],
+  decorators: [PageDecorator],
```

Comment:
I think you'll want to style all your components with the global style, no?

Then, just add it to global decorators like https://github.com/gnosis/safe-cmm-app-react/blob/development/.storybook/preview.js#L9

Reply:
Yep, Dima did that. I talked before with it. He did a PR moving it to the `preview.js`